modules/ROOT/pages/cli-commands-troubleshoot.adoc

```diff
@@ -0,0 +1,102 @@
++
+Here is an example updated command:
++
+----
+anypoint-cli-v4 exchange asset upload my-new-auth-best-practices/1.0.0 --name "My Best Practices Ruleset" --description "This ruleset enforces my best practices for APIs." --files='{"ruleset.yaml":"/myRulesetFolder/mynewruleset.yaml","docs.zip":"/myRulesetFolder/ruleset.doc.zip"}'
+----
+
+[[asset-type-error]]
+== Error Determining Asset Type
+
+The error `Could not determine asset type, step: Publishing asset` when using the `exchange asset upload` command to publish a ruleset or other asset might occur when you:
+
+* Specify a `--type` flag and more than one file type in one or more `--files` flags. 
++
+If you are uploading multiple files, omit the `--type` flag to let Anypoint CLI infer the correct file type from the `classifier.packaging` in each `--files` flag.
+* Specify a `--type` flag that conflicts with the asset type Anypoint CLI infers. 
++
+Verify that the `classifier.packaging` and file name values in your `--files` flag options are correct and that they match the asset type you specified in the `--type` flag. 
+* Include properties in `exchange.json` or `--properties` that are incorrect.
++
+The `exchange.json` file is optional when uploading an asset, because the required properties are either included or are inferred from parameters, flags, or options. If you download a ruleset project on which to base your custom ruleset, do one of the following before uploading your custom ruleset:
++
+** Remove the `exchange.json` file from your working folder or project.
+** Update the properties in the `exchange.json` file to match your custom ruleset's asset ID, name, main file, and so on.
+
+[[generic-publish-error]]
+== Unable to Publish 
+
+A non-specific publishing error similar to `unable to publish` might occur when you're using the `exchange asset upload` command to upload your custom ruleset and its documentation. 
+
+If you attempt to upload an asset using the wrong command syntax, you might inadvertently publish it as the wrong asset type. You cannot republish the asset with the same asset ID as a different asset type. 
+
+To republish the ruleset in this situation, fix the syntax and then change the asset ID and try uploading again. For example, change `my-auth-best-practices` to `my-new auth-best-practices`. 
+
+Here is an example updated command:
+
+----
+anypoint-cli-v4 exchange asset upload my-new-auth-best-practices/1.0.0 --name "My Best Practices Ruleset" --description "This ruleset enforces my best practices for APIs." --files='{"ruleset.yaml":"/myRulesetFolder/mynewruleset.yaml","docs.zip":"/myRulesetFolder/ruleset.doc.zip"}'
+----
+
+[[parse-properties-error]]
+== Error Parsing Properties
+
+When using Anypoint CLI commands that pass JSON properties from Microsoft Windows, you might get an error similar to the following:
+----
```

Comment:
```suggestion

----
```
Not sure if an empty line is needed here, but Github highlighting gets messy after this point. It wouldn't hurt to add it just in case.